backend/Storage.go

```diff
@@ -1,7 +1,68 @@
 package main
 
 // TODO:
-// curl commands for firebase
-// methods to store, publish, modify, delete
 // JSON schema for user profile
 // IM schema and maintenance
+
+import (
+	"context"
+	"errors"
+	"io/ioutil"
+	"fmt"
+	"log"
+	"os"
+
+	firebase "firebase.google.com/go"
+	"gopkg.in/zabawaba99/firego.v1"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/option"
+)
+
+var fbApp     *firebase.App
+var fbClient  *firego.Firebase
+
+func InitializeFirebase() {
+	// This is a Firebase workaround. The Firebase go library MUST read
+	// certificate credentials from a file. Also, Heroku doesn't have static
+	// storage. So, we must create the credential file dynamically from an
+	// environment variable containing the json
+	config := []byte(os.Getenv("FIREBASE_CONFIG"))
```

Comment:
can we not have this file in the backend directory?

Reply:
The file contains a private key that must be loaded directly in order to instantiate Firebase. It is a but of a hacky workaround for now. We could possibly create a .env folder or something to store it though.